src/editors/config.editor.tsx

```diff
@@ -87,8 +86,8 @@ export const HeadersEditor = (props: DataSourcePluginOptionsEditorProps<Infinity
       <Collapse isOpen={true} collapsible={true} label="URL Query Param">
         <SecureFieldsEditor dataSourceConfig={options} onChange={onOptionsChange} title="URL Query Param" secureFieldName="secureQueryName" secureFieldValue="secureQueryValue" hideTile={true} />
       </Collapse>
-      <Collapse isOpen={true} collapsible={true} label="Query Param Encoding (EXPERIMENTAL)">
-        <QueryParamEditor options={options} onOptionsChange={onOptionsChange} />
+      <Collapse isOpen={true} collapsible={true} label="Misc URL settings">
```

Comment:
```suggestion
      <Collapse isOpen={true} collapsible={true} label="URL settings">
```

What do you think about calling this just URL settings? 🤔 I don't think `Misc` adds any additional value, so I believe it could be ommited.